src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,9 +5,19 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+
+    private int paramBull1;
+    private String paramBull2;
```

Comment:
lets create some real parameters like "model", "power" smth like this
same for other classes